app/src/main/res/layout/fragment_report.xml

```diff
@@ -0,0 +1,119 @@
+        android:id="@+id/drawer_layout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        >
+
+        <android.support.design.widget.CoordinatorLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            >
+
+            <android.support.design.widget.AppBarLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                >
+
+                <android.support.v7.widget.Toolbar
+                    android:id="@+id/toolbar"
+                    android:layout_width="match_parent"
+                    android:layout_height="?attr/actionBarSize"
+                    android:background="?attr/colorPrimary"
+                    />
+
+            </android.support.design.widget.AppBarLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="@dimen/dp_5"
+                android:orientation="horizontal"
+                app:layout_behavior="@string/appbar_scrolling_view_behavior"
+                >
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:orientation="horizontal"
+                    android:layout_marginTop="@dimen/dp_1"
+                    android:weightSum="3"
+                    android:layout_marginBottom="@dimen/dp_2"
+                    >
+                    <Button
+                        bind:isPressed="@{viewModel.sale}"
+                        android:layout_marginLeft="@dimen/dp_2"
+                        android:layout_marginStart="@dimen/dp_2"
+                        android:layout_marginRight="@dimen/dp_1"
+                        android:layout_marginEnd="@dimen/dp_2"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
+                        android:background="@drawable/bt_report"
+                        android:text="@string/rp_bt_sales"
+                        android:onClick="@{viewModel::onClickSale}"
+                        />
+                    <Button
+                        bind:isPressed="@{viewModel.customer}"
+                        android:layout_marginLeft="@dimen/dp_1"
+                        android:layout_width="0dp"
+                        android:layout_marginRight="@dimen/dp_1"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
+                        android:background="@drawable/bt_report"
+                        android:text="@string/rp_bt_customers"
+                        android:onClick="@{viewModel::onClickCustomer}"
+                        />
+                    <Button
```

Comment:
WARNING: [ButtonStyle] Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)
 Button bars typically use a borderless style for the buttons. Set the `style="?android:attr/buttonBarButtonStyle"` attribute on each of the buttons, and set `style="?android:attr/buttonBarStyle"` on the parent layout